app/src/main/res/layout/screen_manual_patient_entry.xml

```diff
@@ -334,7 +334,7 @@
             android:id="@+id/colonyOrVillageEditText"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:hint="@string/patiententry_colony_or_village"
```

Comment:
Do we need to manually provide hints anymore? Maybe this can be changed to the `tools:` namespace for editor preview only.

Reply:
There are quite a few text fields whose hint is not manually being set in code. So to be consistent I think it is best to leave these as they are.